docs/dispatcher-install.md

```diff
@@ -72,6 +65,66 @@ Note: ```-auth-version``` is the version of the deployed Openstack Identity Serv
 }
 ```
 
+- The `$noreply_email` and `$noreply_password` will be used by the SAPS deploy to send email notifications, however, in the current version, it is necessary to enable a setting that allows access to the less secure app, so do the following steps:
+
+### Gmail
+
+Sign in to your Gmail account and go to **Manage your Google account**
+![Manage your Google account](img/dispatcher-install-configure-noreply-email-img1.png)
+
+After, access **Security** settings
```

Comment:
Then, navigate to Security settings... I guess it sounds better

Reply:
Done